### PR TITLE
Add issue template

### PR DESCRIPTION
**Pull request by LyzardKing**
@didrocks The github issue template.
This should help most people with an updated version of umake.
Should it say anything different?

### CODE REVIEW

File: ISSUE_TEMPLATE.md [hunk 1]
```diff
@@ -0,0 +1,6 @@
+Make sure you try the following before submitting the issue, thank you!
+
+- Check that umake is the latest version (16.11.1)
```
Comment (didrocks) [inline ISSUE_TEMPLATE.md]: I wouldn't specify any revision here for maintenance purpose but points to https://github.com/ubuntu/ubuntu-make/releases.
Tell they can check the version with `umake --version`

File: ISSUE_TEMPLATE.md [hunk 2]
```diff
@@ -0,0 +1,6 @@
...
+- If you're not running the latest development version of Ubuntu, zesty (17.04), or a distro based on zesty, add the ppa as described in the README.
```
Comment (didrocks) [inline ISSUE_TEMPLATE.md]: Same, don't specify any version (and there is no distro based on development version normally).

File: ISSUE_TEMPLATE.md [hunk 3]
```diff
@@ -0,0 +1,6 @@
...
+
+If all this doesn't solve the problem you're having, please submit the issue including the version of ubuntu and umake.
```
Comment (didrocks) [inline ISSUE_TEMPLATE.md]: s/and//

File: ISSUE_TEMPLATE.md (hunk 1 above)
Comment (LyzardKing) [inline ISSUE_TEMPLATE.md]: It seemed more convenient and harder to ignore if the correct version is there. But sure, I can remove it if you think it's better. There is the problem that at the moment the github releases does not show the latest.

Comment (didrocks) [inline ISSUE_TEMPLATE.md]: it does :) You didn't check in the last 9 minutes this page it seems! ;)

File: ISSUE_TEMPLATE.md (hunk 2 above)
Comment (LyzardKing) [inline ISSUE_TEMPLATE.md]: I was thinking about things like ubuntu-mate, or other belonging to the same ubuntu family...just to be clear.

File: ISSUE_TEMPLATE.md (hunk 3 above)
Comment (LyzardKing) [inline ISSUE_TEMPLATE.md]: what I mean is for instance:
Ubuntu Mate 16.10
umake 16.11.1

So both ubuntu and umake

Comment (LyzardKing) [inline ISSUE_TEMPLATE.md]: what I mean is for instance:
Ubuntu Mate 16.10
umake 16.11.1

So both ubuntu and umake

File: ISSUE_TEMPLATE.md (hunk 1 above)
Comment (LyzardKing) [inline ISSUE_TEMPLATE.md]: ...Magic!

File: ISSUE_TEMPLATE.md (hunk 2 above)
Comment (didrocks) [inline ISSUE_TEMPLATE.md]: ok, the official term is "flavor"

File: ISSUE_TEMPLATE.md (hunk 3 above)
Comment (didrocks) [inline ISSUE_TEMPLATE.md]: ahhhh, maybe… "and umake itself?"

File: ISSUE_TEMPLATE.md (hunk 1 above)
Comment (didrocks) [inline ISSUE_TEMPLATE.md]: lalalala ;-)

File: ISSUE_TEMPLATE.md (hunk 2 above)
Comment (LyzardKing) [inline ISSUE_TEMPLATE.md]: ...and I knew that... silly me

File: ISSUE_TEMPLATE.md (hunk 3 above)
Comment (LyzardKing) [inline ISSUE_TEMPLATE.md]: better

**Comment by didrocks** [comment]
Merging this! Will add one more line afterwards. Many thanks!

**Pull request merged**